geode-protobuf-messages/src/main/proto/v1/clientProtocol.proto

```diff
@@ -75,6 +75,9 @@ message Message {
 
         DisconnectClientRequest disconnectClientRequest = 30;
         DisconnectClientResponse disconnectClientResponse = 31;
+
+        ClearRequest clearRequest = 32;
```

Comment:
I think ClearRegion{Request,Response} would be clearer, but that's a fairly minor quibble.

Reply:
I was following the model of `(Get|Put)Request`.